docs/manual/src/docs/asciidoc/_includes/servlet/oauth2/oauth2-resourceserver.adoc

```diff
@@ -1258,6 +1258,29 @@ http
 Resolving the tenant by claim is similar to doing so by request material.
 The only real difference is the `toTenant` method implementation:
 
+[source,java]
+----
+@Component
+public class TenantRepository {
```

Comment:
I see your point in adding this sample `TenantRepository`. But, I think this demonstrates actually that it would be better to change the classes below to simply use a pair of `Map`s.

Would you mind changing the examples to use a `Map<String, String>` instead? Then, the members might be called `issuersByTenant` and `jwkUrisByTenant` instead of `tenants` or `tenantRepository`.